lib/ash/can.ex

```diff
@@ -1,8 +1,28 @@
 defmodule Ash.Can do
-  @moduledoc false
+  @moduledoc """
+  Contains the Ash.can function logic.
+  """
 
   require Ash.Query
 
+  @type query_or_changeset_or_action ::
```

Comment:
Let's call this `subject`, as opposed to `query_or_changeset_or_action`. Great change though 🔥